CMakeLists.txt

```diff
@@ -51,15 +51,12 @@ endif(NOT DEFINED ENV{KRATOS_INSTALL_PYTHON_USING_LINKS})
 set(CMAKE_INSTALL_RPATH "$ORIGIN/../libs")
 
 # If no test policy enable by default
-if(NOT DEFINED KRATOS_BUILD_TESTING)
-message("-- No KRATOS_BUILD_TESTING is defined. Setting to ON")
-SET (KRATOS_BUILD_TESTING ON)
-endif(NOT DEFINED KRATOS_BUILD_TESTING)
+option(KRATOS_BUILD_TESTING "-- No KRATOS_BUILD_TESTING is defined. Setting to ON" ON)
```

Comment:
This is not the same, the string is the name of the option, but it will now no longer issue the message

If you use it then I would improve the message

Reply:
In the doc only speaks about <help_text>, I don't know how it was defined

Reply:
@philbucher Oh you are right